test/e2e-test-application/externalMf/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "luigi-external-mf",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.html",
+  "license": "ISC",
```

Guess it should also be MIT, since everything is mit.
```suggestion
  "license": "MIT",
```